data/FilterList.json

```diff
@@ -7408,5 +7408,59 @@
     "submissionUrl": null,
     "syntaxId": 3,
     "viewUrl": "https://raw.githubusercontent.com/PolishFiltersTeam/PolishRSSFilters/master/polish_rss_filters.txt"
+  },
+  {
+    "id": 391,
+    "description": "Removes popular games and most bad games from the front pages of various webstores and app stores.",
+    "descriptionSourceUrl": null,
+    "discontinuedDate": null,
+    "donateUrl": "https://sproutsluckycorner.wordpress.com/2017/11/14/my-work-and-contact-resume/#donations",
+    "emailAddress": "imreeil42@gmail.com",
+    "forumUrl": null,
+    "homeUrl": "https://github.com/DandelionSprout/adfilt",
+    "issuesUrl": "https://github.com/DandelionSprout/adfilt/issues",
+    "licenseId": 10,
+    "name": "I Hate Overpromoted Games",
+    "policyUrl": null,
+    "publishedDate": null,
+    "submissionUrl": null,
+    "syntaxId": 3,
+    "viewUrl": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/IHateOverpromotedGames.txt"
+  },
+  {
+    "id": 392,
+    "description": "Intends to make it easier to browse various websites without being forced by various nag-windows to log in. At the time of writing (15th of May 2018) it chiefly supports Pinterest, Facebook and Pixiv.",
+    "descriptionSourceUrl": null,
+    "discontinuedDate": null,
+    "donateUrl": "https://sproutsluckycorner.wordpress.com/2017/11/14/my-work-and-contact-resume/#donations",
+    "emailAddress": "imreeil42@gmail.com",
+    "forumUrl": null,
+    "homeUrl": "https://github.com/DandelionSprout/adfilt",
+    "issuesUrl": "https://github.com/DandelionSprout/adfilt/issues",
+    "licenseId": 10,
+    "name": "Browsing websites without logging in",
+    "policyUrl": null,
+    "publishedDate": "2018-05-15T00:00:00",
+    "submissionUrl": null,
+    "syntaxId": 3,
+    "viewUrl": "https://raw.githubusercontent.com/DandelionSprout/adfilt/master/BrowseWebsitesWithoutLoggingIn.txt"
+  },
+  {
+    "id": 393,
+    "description": "A list that demonstrates and makes use of Einar Egilsson's Redirector extensions, in order to accomplish things that no adblock tool could've achieved. Examples include turning 32-bit Windows program installation links into 64-bit ones, making Disney's US websites accessible from other countries, and turning mobile Wikipedia links into desktop ones, among other things.",
+    "descriptionSourceUrl": null,
+    "discontinuedDate": null,
+    "donateUrl": "https://sproutsluckycorner.wordpress.com/2017/11/14/my-work-and-contact-resume/#donations",
+    "emailAddress": "imreeil42@gmail.com",
+    "forumUrl": null,
+    "homeUrl": "https://github.com/DandelionSprout/adfilt/tree/master/Dandelion%20Sprout-s%20Redirector%20Assistant%20List",
+    "issuesUrl": "https://github.com/DandelionSprout/adfilt/issues",
+    "licenseId": 10,
+    "name": "Dandelion Sprout's Redirector List",
+    "policyUrl": null,
+    "publishedDate": "2018-05-15T00:00:00",
+    "submissionUrl": null,
+    "syntaxId": null,
```

Comment:
Hmm, interesting. hadn't heard of the Redirector extension. Cool. Does it make sense to add a new syntax that this extension supports? If so, please feel free to add a new element to the Syntax json.